config/environments/production.rb

```diff
@@ -51,8 +51,7 @@
 
   # Replace the default in-process and non-durable queuing backend for Active Job.
   config.active_job.queue_adapter = :solid_queue
-  config.solid_queue.connects_to = { database: { writing: :queue } }
-
+  config.solid_queue.connects_to = { database: { writing: :production } }
```

Comment:
same as above

Reply:
I thought we were going the path of using a separate database for solid queue. I'm in the process of (trying to) set that up. It would be called queue, the default.

Reply:
Yeah, this was only to get the deploy to work until we have them. If they will be ready before we want to merge this, I can change it ASAP.